pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go

```diff
@@ -1,20 +1,19 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright The Kubernetes Authors.
+Copyright 2018 Weaveworks. All rights reserved.
```

Comment:
@errordeveloper @2opremio Is this change ok? seems to be done automatically now.

Reply:
Yes, that's fine.